src/components/App.tsx

```diff
@@ -666,13 +661,15 @@ export class App extends React.Component<AppProps, AppState> {
     </Split>;
 
     return <div className="fill">
-      {this.state.toast &&
+      <div className="toast-container">
+      {this.state.toasts && this.state.toasts.map((toast: string, key: number) =>
```

Comment:
What about other kinds of toasts? I think of toasts as notifying the user of events that they may need to take an action on. This seems hardcoded to Gist toasts.

Reply:
Also, you probably want to check the length of the toasts, not just the presence of the array.